source/common/upstream/cluster_manager_impl.cc

```diff
@@ -2214,23 +2216,25 @@ void ClusterManagerImpl::ThreadLocalClusterManagerImpl::httpConnPoolIsIdle(
   }
 }
 
-HostConstSharedPtr ClusterManagerImpl::ThreadLocalClusterManagerImpl::ClusterEntry::chooseHost(
+HostSelectionResponse ClusterManagerImpl::ThreadLocalClusterManagerImpl::ClusterEntry::chooseHost(
     LoadBalancerContext* context) {
   auto cross_priority_host_map = priority_set_.crossPriorityHostMap();
   HostConstSharedPtr host = HostUtility::selectOverrideHost(cross_priority_host_map.get(),
                                                             override_host_statuses_, context);
   if (host != nullptr) {
-    return host;
+    return {std::move(host), nullptr};
```

Comment:
Given that nullptr is the default, I think this could probably be omitted. But of course it's fine to specify it explicitly.

Reply:
yeah I added the default after seeing how many more edits it created.  cleaned this one up :-)